geth/common/utils.go

```diff
@@ -143,7 +131,7 @@ func Fatalf(reason interface{}, args ...interface{}) {
 	// find out whether error or string has been passed as a reason
 	r := reflect.ValueOf(reason)
 	if r.Kind() == reflect.String {
-		fmt.Fprintf(w, "Fatal Failure: "+reason.(string)+"\n", args) //nolint: gas
+		fmt.Fprintf(w, "Fatal Failure: %v\n%v", reason.(string), args) //nolint: gas
```

Comment:
One tiny thing that is also not your fault, but it is nice to fix. We probably need to add `\n` after the second `%v`. Otherwise, the output of this log entry will overflow the next one.